_includes/tinymce-script-tag.html

```diff
@@ -1,2 +1,2 @@
-<script src='{{ site.tinymce_codepen_url }}' referrerpolicy="origin"></script>
+<script src='{{ site.tinymce_live-demo_url }}' referrerpolicy="origin"></script>
```

Comment:
Some more inconsistent naming here.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/50d665fdbcdb9afd3395dccf0ff6eba409f4895d